package.json

```diff
@@ -269,11 +278,22 @@
         {
           "command": "terraform.modules.openDocumentation",
           "when": "view == terraform.modules"
+        },
+        {
+          "command": "terraform.providers.openDocumentation",
+          "when": "view == terraform.providers && viewItem == moduleProviderHasDocs"
         }
       ]
     },
     "views": {
       "explorer": [
+        {
+          "id": "terraform.providers",
+          "name": "Terraform Provides",
```

Comment:
Is `Terraform Provides` a typo or intentional? I expected this to be `Providers`

Reply:
Typo, good catch!

Reply:
I know the repetition of "Terraform" through both of our codebases is a common problem 😅  but I would argue that it may actually be relevant here as this is the title of the pane within the bigger otherwise not extension specific Explorer view.

I guess it may be obvious what kind of "providers" are these to most users, but we are sharing this space with other extensions in principle and so I'd be careful about the naming here.

Reply:
I agree @radeksimko, I didn't mean to imply we remove the `Terraform` part. Meant just to call out the typo